src/compiler/checker.ts

```diff
@@ -39998,7 +39998,7 @@ namespace ts {
 
         function isAliasResolvedToValue(symbol: Symbol): boolean {
             const target = resolveAlias(symbol);
-            if (target === unknownSymbol) {
+            if (target === unknownSymbol || (target.flags & SymbolFlags.ExportValue)) {

```

Comment:
(part of the previous suggestion, since GitHub won't allow me to create a suggestion that covers both lines due to the deleted line).
```suggestion
            if (target === unknownSymbol) {
```